packages/jsii-dotnet-runtime/src/Amazon.JSII.Runtime/Deputy/JsiiTypeAttributeBase.cs

```diff
@@ -1,14 +1,17 @@
-﻿using Amazon.JSII.Runtime.Services;
-using Microsoft.Extensions.DependencyInjection;
-using System;
-using System.Collections.Generic;
+﻿using System;
+using System.Collections.Concurrent;
 using System.Linq;
 using System.Reflection;
+using Amazon.JSII.Runtime.Services;
+using Microsoft.Extensions.DependencyInjection;
 
 namespace Amazon.JSII.Runtime.Deputy
 {
     public abstract class JsiiTypeAttributeBase : Attribute
     {
+        private static readonly ConcurrentBag<string> ProcessedAssemblies =
```

Comment:
Nit: Are we still attempting to follow [the CoreFX style guide](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md)?

> We use \_camelCase for internal and private fields and use readonly where possible. Prefix internal and private instance fields with \_, static fields with s\_ and thread static fields with t\_. When used on static fields, readonly should come after static (e.g. static readonly not readonly static). Public fields should be used sparingly and should use PascalCasing with no prefix when used.

I don't like the Hungarian notation, we can omit that. But PascalCase is reserved for public fields.

Reply:
We are still attempting this. But since it is a static readonly, it follows -kinda- the same rules as a const, so it's PascalCase. If it was not readonly, it would be camelCase.

I could change my mind though. I'm following the guidance of ReSharper here.